test/minikube/minikube_kaa_additions_test.go

```diff
@@ -92,3 +92,31 @@ func TestKaaLimitedPermissions(t *testing.T) {
 	require.True(t, len(config.Volumes) == 0)
 	require.True(t, len(config.Pods) == 0)
 }
+
+func TestKaaRolebindingDisabled(t *testing.T) {
+	// This test requires NuoDB 4.2+ or 4.1.2+
```

Comment:
This file is a convenient place to put the test for now that the functionality is unreleased and can only be tested internally, but it does not require an Enterprise license, and we should put it in a different file once the 4.1.2/4.2 are released so that this can run in Travis.

Before tests like these pile up that are candidates for Travis, we should create a new convention for testing unreleased functionality that is separate from the `enterprise` build tag. We should put tests like these in a normal test file (not having the `enterprise` tag) and explicitly check an environment variable like `UNRELEASED_FUNCTIONALITY`. And then we can remove the check whenever a relevant release is made.